docs/book/mlops_stacks/metadata_stores/custom.md

```diff
@@ -0,0 +1,57 @@
+---
+description: Extend ZenML to implement a custom Metadata Store
+---
+
+{% hint style="warning" %}
+**Base abstraction in progress!**
+
+We are actively working on the base abstraction for the Metadata Stores, which 
+will be available soon. As a result, their extension is not recommended at the 
```

Comment:
I wouldnt make that promise. I dont think its coming soon. We should say that its not available.